test/common/gristUrls.ts

```diff
@@ -64,4 +64,44 @@ describe('gristUrls', function() {
       assert.deepEqual(parseFirstUrlPart('o', ''), {path: ''});
     });
   });
+
+  describe('getHostType', function() {
+    const defaultOptions = {
+      baseDomain: 'getgrist.com',
+      pluginUrl: 'https://plugin.getgrist.com/path',
```

Comment:
Why the extra path at the end? I think Grist assumes (silently?) that this is an absolute URL (protocol + host) only. At least I never tested or see it being used with a path.

Reply:
Thanks for your feedback!

I am not sure of the purpose of this variable (nor of `APP_UNTRUSTED_URL`), so I inferred that test from what I saw in the code. 

From what I see in your question, I will remove the path and only keep the protocol + host.